code/modules/projectiles/ammunition/energy/stun.dm

```diff
@@ -1,19 +1,22 @@
 /obj/item/ammo_casing/energy/electrode
 	projectile_type = /obj/projectile/energy/electrode
-	select_name = "stun"
+	select_name = "tasing"
 	fire_sound = 'sound/weapons/taser.ogg'
-	e_cost = 200
+	e_cost = 100
 	harmful = FALSE
 
+/obj/item/ammo_casing/energy/electrode/turret
+	projectile_type = /obj/projectile/energy/electrode/turret
+	select_name = "stun"
+
 /obj/item/ammo_casing/energy/electrode/spec
-	e_cost = 100
+	e_cost = 50
 
 /obj/item/ammo_casing/energy/electrode/gun
 	fire_sound = 'sound/weapons/gun/pistol/shot.ogg'
-	e_cost = 100
 
 /obj/item/ammo_casing/energy/electrode/old
-	e_cost = 1000
+	e_cost = 250

```

Comment:
Im conflicted on this. Why is it needed?

Reply:
because tasers no longer stun instantly, ruining most of the point of giving whatever gun this is a single shot. giving it 4 shots is a lot more fair as 2 or 3 shots can hit limbs, vastly reducing overall stamina damage

Reply:
With four shots this gets a lot stronger than it used to be. Limb shots are rare-ish, Im not sure the justification is good enough. I believe the gun in question is said to have an experimental taser mode, so it being a bit shit is fine.